content/docs/ops/maintenance-list.md

```diff
@@ -29,7 +29,7 @@ that week.
 
 - Confirm that all AWS users have MFA configured. The following should return `[]`
 or the users that don't have MFA enabled:
-```shell
+```bash
```

Comment:
+1 I always err on the side of `sh` and trying to keep things posix compliant so it's truly portable.